packages/widgets/src/blocks/legacy-widget/edit/preview.js

```diff
@@ -96,12 +130,7 @@ export default function Preview( { idBase, instance, isVisible } ) {
 						// TODO: This chokes when the query param is too big.
 						// Ideally, we'd render a <ServerSideRender>. Maybe by
 						// rendering one in an iframe via a portal.
```

Comment:
I believe we can remove this comment now?

Reply:
Fixed in 98d0fafc27ee1bb29952446a6bcc394b7c510ec1